specification/servicefabric/resource-manager/Microsoft.ServiceFabric/preview/2023-11-01-preview/cluster.json

```diff
@@ -0,0 +1,2167 @@
+        "operationId": "Operations_List",
+        "parameters": [
+          {
+            "name": "api-version",
+            "in": "query",
+            "description": "The version of the Service Fabric resource provider API",
+            "required": true,
+            "type": "string"
+          }
+        ],
+        "x-ms-examples": {
+          "List available operations": {
+            "$ref": "./examples/Operations_example.json"
+          }
+        },
+        "responses": {
+          "200": {
+            "description": "OK. The request has succeeded.",
+            "schema": {
+              "$ref": "#/definitions/OperationListResult"
+            }
+          },
+          "default": {
+            "description": "Error response describing why the operation failed.",
+            "schema": {
+              "$ref": "#/definitions/ErrorModel"
+            }
+          }
+        },
+        "x-ms-pageable": {
+          "nextLinkName": "nextLink"
+        }
+      }
+    },
+    "/subscriptions/{subscriptionId}/providers/Microsoft.ServiceFabric/locations/{location}/unsupportedVmSizes": {
+      "get": {
+        "tags": [
+          "VMSizes"
+        ],
+        "summary": "Get the lists of unsupported vm sizes for Service Fabric Clusters.",
+        "description": "Get the lists of unsupported vm sizes for Service Fabric Clusters.",
+        "operationId": "UnsupportedVmSizes_List",
+        "parameters": [
+          {
+            "$ref": "#/parameters/locationForClusterCodeVersions"
+          },
+          {
+            "$ref": "#/parameters/api-version"
+          },
+          {
+            "$ref": "#/parameters/subscriptionId"
+          }
+        ],
+        "x-ms-examples": {
+          "List unsupported vm sizes": {
+            "$ref": "./examples/UnsupportedVMSizesList_example.json"
+          }
+        },
+        "responses": {
+          "200": {
+            "description": "The operation completed successfully.",
+            "schema": {
+              "$ref": "#/definitions/VMSizesResult"
+            }
+          },
+          "default": {
+            "description": "The detailed error response.",
+            "schema": {
+              "$ref": "#/definitions/ErrorModel"
+            }
+          }
+        },
+        "x-ms-pageable": {
+          "nextLinkName": "nextLink"
+        }
+      }
+    },
+    "/subscriptions/{subscriptionId}/providers/Microsoft.ServiceFabric/locations/{location}/unsupportedVmSizes/{vmSize}": {
+      "get": {
+        "tags": [
+          "VMSizes"
+        ],
+        "summary": "Get unsupported vm size for Service Fabric Clusters.",
+        "description": "Get unsupported vm size for Service Fabric Clusters.",
+        "operationId": "UnsupportedVmSizes_Get",
+        "parameters": [
+          {
+            "$ref": "#/parameters/locationForClusterCodeVersions"
+          },
+          {
+            "$ref": "#/parameters/api-version"
```

Comment:
>" [](http://example.com/codeflow?start=45&length=1)

please fix the newly introduced warnings here : [https://github.com/Azure/azure-rest-api-specs/pull/26671/checks?check_run_id=18796589308](https://github.com/Azure/azure-rest-api-specs/pull/26671/checks?check_run_id=18796589308)

Reply:
For common-types we have work planned for this https://msazure.visualstudio.com/One/_workitems/edit/25890469, But it's going to take some time because we generate the swagger from an internal repo that currently can't reference the common types.